sql/src/main/java/io/crate/metadata/settings/LoggingSetting.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.metadata.settings;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.ImmutableSet;
+import org.apache.commons.lang3.StringUtils;
+
+import javax.annotation.Nullable;
+import java.util.Locale;
+import java.util.Set;
+
+public class LoggingSetting extends StringSetting {
+
+    private static final Set<String> LOGGING_VALUES = ImmutableSet.of("TRACE", "DEBUG", "INFO", "WARN", "ERROR", "FATAL");
+
+    public LoggingSetting(String name) {
+        super(name, LOGGING_VALUES, true);
+    }
+
+    @Nullable
+    public String validate(String value) {
+        for (String allowedValue : allowedValues) {
+            if (StringUtils.equalsIgnoreCase(allowedValue, value)) {
```

Comment:
please use `allowedValue.equalsIgnoreCase(value)`  no need for third party dependency for this and the null check isn't necessary.